ta/tpm_log_test/include/user_ta_header_defines.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright (c) 2020, ARM Limited. All rights reserved.
+ */
+
+#ifndef USER_TA_HEADER_DEFINES_H
+#define USER_TA_HEADER_DEFINES_H
+
+#include <ta_tpm_log.h>
+
+#define TA_UUID TA_TPM_LOG_TEST_UUID
+
+#define TA_FLAGS		(TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR)
```

Comment:
Both flags are deprecated, please use `0` instead (yes a lot of code would need cleaning up ;-)).

Reply:
I noticed that TA_FLAGS is not used, should I just remove it instead of making it 0?

Reply:
It is used in [`user_ta_header.c`](https://github.com/OP-TEE/optee_os/blob/3.8.0/ta/arch/arm/user_ta_header.c#L83) which is imported as part of the TA build.

But my suggestion to use zero is not good, see [here](https://github.com/linaro-swg/optee_examples/pull/69). For consistency I'd like to see `TA_FLAG_EXEC_DDR` here.